deploy.sh

```diff
@@ -76,7 +76,7 @@ rm -rf $SVNPATH/assets/*
 
 echo "Exporting the HEAD of master from git to the trunk of SVN"
 git checkout-index -a -f --prefix=$SVNPATH/trunk/
-git checkout-index -a -f --prefix=$SVNPATH/ assets/*
+git checkout-index -f --prefix=$SVNPATH/ assets/*
```

Comment:
Should there be a space between `$SVNPATH/  and `assets/*` ?

Reply:
Yep, `$SVNPATH/` is the `prefix` option value and `assets/*` are the files to copy.